docs/user-guides/benchmark.rst

```diff
@@ -0,0 +1,294 @@
++----------------------------------+------------+-----------+------------+-----------+-----------+------------+-----------+-----------+-----------+-----------+-----------+----------------+-------------+-------------+----------------+-------------------+----------------------+-------------------+---------------------+-----------+----------------+
+| ViT-L-14-336::openai             | **0.709**  | 0.781     | 0.837      | 0.744     | 0.556     | 0.783      | 0.937     | 0.940     | 0.560     | 0.615     | 0.638     | 0.608          | 0.733       | 0.200       | 0.158          | 0.032             | 0.024                | 0.046             | 0.113               | 0.158     | 0.262          |
++----------------------------------+------------+-----------+------------+-----------+-----------+------------+-----------+-----------+-----------+-----------+-----------+----------------+-------------+-------------+----------------+-------------------+----------------------+-------------------+---------------------+-----------+----------------+
+| ViT-H-14::laion2b_s32b_b79k      | **0.709**  | 0.777     | 0.850      | **0.847** | 0.678     | **0.801**  | **0.945** | 0.961     | 0.563     | **0.726** | 0.699     | 0.542          | 0.297       | 0.268       | 0.169          | 0.032             | 0.027                | 0.054             | 0.111               | 0.140     | 0.110          |
++----------------------------------+------------+-----------+------------+-----------+-----------+------------+-----------+-----------+-----------+-----------+-----------+----------------+-------------+-------------+----------------+-------------------+----------------------+-------------------+---------------------+-----------+----------------+
+| ViT-g-14::laion2b_s12b_b42k      | 0.696      | **0.811** | **0.851**  | 0.839     | **0.682** | 0.776      | 0.943     | **0.962** | **0.603** | 0.648     | 0.718     | 0.560          | 0.580       | **0.332**   | 0.175          | 0.036             | 0.031                | 0.060             | 0.115               | 0.190     | 0.138          |
++----------------------------------+------------+-----------+------------+-----------+-----------+------------+-----------+-----------+-----------+-----------+-----------+----------------+-------------+-------------+----------------+-------------------+----------------------+-------------------+---------------------+-----------+----------------+
+
+From the table, we observe that the ViT models still outperform the RN models in most tasks, except for the Patch Camelyon dataset where ``RN50::openai`` has the best top-1 accuracy of 0.636, and the KITTI/distance dataset where ``RN50::yfcc15m`` has the best result of 0.336.
```

Comment:
There are some cases where RN models beat ViT, such as `Patch Camelyon`, `KITTI/distance`. Is there any reason that they perform like this way?